src/bignum.cc

```diff
@@ -0,0 +1,165 @@
+// Copyright (C) 2022 Toitware ApS.
+//
+// This library is free software; you can redistribute it and/or
+// modify it under the terms of the GNU Lesser General Public
+// License as published by the Free Software Foundation; version
+// 2.1 only.
+//
+// This library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+// Lesser General Public License for more details.
+//
+// The license can be found in the file `LICENSE` in the top level
+// directory of this repository.
+
+#include "top.h"
+#include "objects_inline.h"
+#include "process.h"
+#include "resource.h"
+#include "resource_pool.h"
+#include "vm.h"
+
+#include "mbedtls/bignum.h"
+
+namespace toit {
+
+typedef int (*mpi_basic_func_t)(mbedtls_mpi *X, const mbedtls_mpi *A, const mbedtls_mpi *B);
```

Comment:
Please use lower case for argument names throughout.

Asterisk is placed with the type, not with the variable throughout.